Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.yml

```diff
@@ -39,3 +41,4 @@ type: python
 fromversion: 6.5.0
 tests:
 - No tests (auto formatted)
+runonce: false
```

Comment:
Could you elaborate on why this change is needed?

Reply:
not sure how/why that was added.  Will remove

Reply:
change in [review updates](https://github.com/demisto/content/pull/29311/commits/d0b01a94de4508529522275932721c124c094d62)